ui/src/pages/frontsite/components/ILSMenu/ILSMenu.js

```diff
@@ -1,22 +1,38 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import { Container, Menu, Dropdown, Responsive } from 'semantic-ui-react';
-import {
-  authenticationService,
-  sessionManager,
-} from '@authentication/services';
+import { authenticationService } from '@authentication/services';
 import { FrontSiteRoutes, BackOfficeRoutes } from '@routes/urls';
-import { LoginButton } from '@components';
+import { RedirectToLoginButton } from '@authentication/components';
+import { goTo } from '@history';
 
 export default class ILSMenu extends Component {
+  logout = async () => {
+    try {
+      const resp = await authenticationService.logout();
+      if (resp) {
+        this.props.fetchhUserProfile();
```

Comment:
double `hh` in `fetchh`. By the way, you are doing a fetchUserProfile to set the app anonymous?
I would rather do a `this.props.setAnonymous` instead, it is not very clear otherwise.